zone/entity.cpp

```diff
@@ -1595,7 +1595,7 @@ void EntityList::RefreshClientXTargets(Client *c)
 }
 
 void EntityList::QueueClientsByTarget(Mob *sender, const EQApplicationPacket *app,
-		bool iSendToSender, Mob *SkipThisMob, bool ackreq, bool HoTT, uint32 ClientVersionBits, bool inspect_buffs)
+		bool iSendToSender, Mob *SkipThisMob, bool ackreq, bool HoTT, uint32 ClientVersionBits, bool inspect_buffs, bool clear)
```

Comment:
`clear` - Clear what? This should read it's intent

Reply:
Yeah, I tend to agree.  This QueueClientsByTarget function kinda sucks and it isn't gonna read clearly no matter what because the internals and one of the arguments are non-generic but the function wants to think it is.  But I updated the variable a bit.

Reply:
The code is certainly messy, but at least the one tweak should help